### ISSUE TITLE
Support -vcolumns flag in DMD 2.066

### ISSUE THREAD

**Issue by nordlow** [flycheck/flycheck#460]
The following activates the -vcolumns and its related parsing if DMD is of version >= 2.066.

``` Lisp
(defun dmd-support-columns (&optional dmd-compiler)
  "DMD 2.066+ has -vcolumns flag."
  (string-match "-vcolumns" (shell-command-to-string "dmd --help")))

(when (dmd-support-columns)
    (flycheck-define-checker d-dmd      ;TODO: Integrate this with flycheck.el
      "A D syntax checker using the DMD compiler.

See URL `http://dlang.org/'."
      :command ("dmd" "-vcolumns" "-debug" "-o-"
                "-wi"     ; Compilation will continue even if there are warnings
                (eval (concat "-I" (flycheck-d-base-directory)))
                ;;(option-list "-I" flycheck-dmd-include-path s-prepend)
                (option-list "-I" flycheck-dmd-include-path concat)
                source)
      :error-patterns
      ((error line-start (file-name) "(" line "," column "): Error: " (message) line-end)
       (warning line-start (file-name) "(" line "," column "): "
                (or "Warning"
                    "Deprecation") ": " (message) line-end))
      :modes d-mode))
```

How do you want to proceed with this?
